scrapy/core/scheduler.py

```diff
@@ -13,9 +12,11 @@
 
 class Scheduler(object):
 
-    def __init__(self, dupefilter, jobdir=None, dqclass=None, mqclass=None, logunser=False, stats=None):
+    def __init__(self, dupefilter, jobdir=None, logunser=False, stats=None,
+                 pqclass=None, dqclass=None, mqclass=None):
```

Comment:
Would it help if I pass the class instancing args in from_crawler as kwargs; so it might be re-ordered in the distant future? Or won't that happen anyway. (as `dqclass=dqclass` and so on)